Forms/MainForm.cs

```diff
@@ -27,6 +28,10 @@ public partial class MainForm : Form
         public ItemEditForm ItemEditor { get; private set; }
         public StartingItemEditForm StartingItemEditor { get; private set; }
 
+        public readonly string ROOT = Path.GetDirectoryName(System.Reflection.Assembly.GetEntryAssembly().Location) + "\\";
```

Comment:
I'm pretty sure if you want to read/write in the same folder that the exe is in, you can just use a relative path. So this is not necessary.